web/themes/new_weather_theme/templates/layout/page--point.html.twig

```diff
@@ -53,26 +52,17 @@
           {{ drupal_block("weathergov_local_alerts") }}
         </div>
         {% endif %}
+
         <div class="tab-container focus-offset-205" id="current" role="tabpanel" aria-labelledby="current-conditions-tab-button" tabindex="0">
           {{ drupal_block("weathergov_current_conditions") }}
           {% include '@new_weather_theme/partials/radar.html.twig' with { 'point': weather.point  } %}
         </div>
-        <div class="tab-container focus-offset-205" id="today" role="tabpanel" aria-labelledby="today-tab-button" tabindex="0">
+
+         <div class="tab-container focus-offset-205" id="daily" role="tabpanel" aria-labelledby="daily-tab-button" tabindex="0">
           {{ drupal_block("weathergov_weather_story") }}
-          <div class="grid-container padding-x-2">
-            <div class="grid-row">
-              <div class="grid-col tablet-lg:grid-offset-2">
-                <h2 class="text-normal text-primary-darkest margin-top-0 margin-bottom-2">{{ "Hourly" | t }}</h2>
-              </div>
-            </div>
-          </div>
-          {{ drupal_block("weathergov_hourly_precipitation") }}
-          {{ drupal_block("weathergov_hourly_forecast") }}
```

Comment:
This hourly_forecast block has  ` {{ attach_library('new_weather_theme/hourly-table') }}`, since we are removing it, the `HourlyTable.js` is no longer being loaded so I think that's causing the buttons to break.

Reply:
Good find!